src/exm-extension-row.blp

```diff
@@ -167,10 +167,13 @@ template $ExmExtensionRow : Adw.ExpanderRow {
 	Gtk.ListBoxRow {
 		activatable: false;
 		focusable: false;
+		height-request: 46;
```

Comment:
This doesn't really play nice with the big text accessibility setting, but it just goes back to the current unpadded state so I think it's fine.